routes/aiRoutes.js

```diff
@@ -1,8 +1,72 @@
+/* eslint-disable indent */
 const express = require('express');
-const { spawn } = require('child_process');
 const multer = require('multer');
 const router = express.Router();
 const { shorttermLimiter, longtermLimiter } = require('../middlewares/rate_limiting');
+const FeedbackAi = require('../models/FeedbackAi');
+const executePythonScript = require('../helpers/excutePython');
+
+
+//get all feedbacks
+router.get('/feedbacks', shorttermLimiter, longtermLimiter, async (req, res) => {
+	try {
+		const feedbacks = await FeedbackAi.find();
+		res.json(feedbacks);
+	} catch (error){
+		console.error('error fetching feedbacks', error.message);
+		res.status(500).json({error: 'Internal server error'});
+	}
+});
+
+router.get('/feedback', shorttermLimiter, longtermLimiter, async (req, res) => {
+    console.log('GET request received at /api/ai/feedback');
+	res.send('feedback route is working!!!???!');
```

Comment:
:O

Reply:
den her er der for testing af endpoints, men kan godt slettes hvis du synes det er dumt at have det der